src/services/CommentService.js

```diff
@@ -12,11 +12,20 @@ const apiClient = axios.create({
 export default {
   async getSolutionComments(solutionId, pageNumber, pageSize) {
     console.log("TEMP API CALL COMMENTS: ", solutionId, pageNumber, pageSize);
-    return await apiClient.get("/Comments").then((response) => response.data);
+    return await apiClient
+      .get(
+        "/comments/solutions/" +
```

Comment:
Wat is deze query params?!?!?¿¿¿ 🤷‍♂️

Reply:
is niet het mooist maar het werkt

Reply:
.get("/comments/solutions/" + solutionId, {
    params: {
        pageNumber: pageNumber,
        pageSize: pageSize
    }
})

Heb t naar dit geze
![MuaKissGIF](https://user-images.githubusercontent.com/47415892/144831435-32166b06-a5a0-404e-9fb7-79d998f229c0.gif)

Reply:
okii